tremor-cli/tests/integration/tcp/config.troy

```diff
@@ -23,7 +25,21 @@ flow
   create connector stdio from connectors::console;
   create connector tcp_server;
 
-  create pipeline server_side from pipelines::passthrough;
+  define pipeline server_side
+  pipeline
+    define script scr
```

Comment:
note for later:

having scripts top set the different connector metadata in a library would be nice in addition to the functions

i.e.

```
use tremor::traints::socket_server;

create script socket_server::set_handle with handle = "output" end;
```